src/components/Menu.tsx

```diff
@@ -14,11 +14,19 @@ import { css } from '../../styled-system/css'
 import { useSupabase } from '@/components/SupabaseProvider'
 import Link from 'next/link'
 import Image from 'next/image'
+import { useInterval } from 'usehooks-ts'
+import { useUpdateUserConnection } from '@/app/domains/profiles/profiles.helpers'
 
 export default function Menu() {
   const { isOpen, onOpen, onClose } = useDisclosure()
   const { session, supabase } = useSupabase()
 
+  const { updateActiveTime } = useUpdateUserConnection()
+
+  useInterval(() => {
```

Comment:
suggestion: Why isn't this part of the hook, why did you extract it?

Reply:
besides it is always striking for me to have a hook that doesn't return anything feels so wrong for some reason 😆

Reply:
To be honest I've never realized that undefined return 😅